docs/configuration.md

```diff
@@ -333,3 +333,26 @@ Credential: "git:https://bob@github.com/example/myrepo" (user = bob)
 
    https://bob@github.com/example/myrepo
 ```
+
+---
+
+### credential.azreposCredentialType _(experimental)_
```

Comment:
hm. this is different than what I had configured. it must have changed. I'll update and re-install. Do you have an updated installer link?

Reply:
Not yet! Will send one over shortly.